instrumentation/spring/README.md

```diff
@@ -12,6 +12,14 @@ The [third section](#auto-instrumentation-using-spring-starters) with build on t
 
 In this guide we will be using a running example. In section one and two, we will create two spring web services using Spring Boot. We will then trace requests between these services using two different approaches. Finally, in section three we will explore tools documented in [opentelemetry-spring-boot-autoconfigure](/spring-boot-autoconfigure/README.md#features) which can improve this process.
 
+# Settings 
+
+| System property | Type | Default | Description |
+|---|---|---|---|
+| `otel.instrumentation.spring-integration.global-channel-interceptor-patterns` | List | `*` | An array of Spring channel name patterns that will be intercepted. See [Spring Integration docs](https://docs.spring.io/spring-integration/reference/html/channel.html#global-channel-configuration-interceptors) for more details. |
+| `otel.instrumentation.spring-webflux.experimental-span-attributes` | Boolean | `false` | Enable the capture of experimental span attributes for Spring WebFlux version 5.0. |
+| `otel.instrumentation.spring-webmvc.experimental-span-attributes` | Boolean | `false` | Enable the capture of experimental span attributes for Sprinv Web MVC 3.1. |
+
```

Comment:
this README.md is all about non-javaagent usage

it has three parts:
* [Manual Instrumentation with Java SDK](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/spring#manual-instrumentation-with-java-sdk)
* [Manual Instrumentation using Handlers and Filters](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/spring#manual-instrumentation-using-handlers-and-filters)
* [Auto Instrumentation using Spring Starters](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/spring#auto-instrumentation-using-spring-starters)

the first two parts aren't specific to any of the instrumentation modules in this repo, and just describe manually instrumenting Spring using OpenTelemetry SDK directly. These two sections may not belong in this repo at all.

the third part could then be moved to `instrumentation/spring/starters`

I'll create an issue to discuss/track this, and note that we would want to keep this section here if we restructure

Reply:
#5002